.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
```

Comment:
How does this differ than the built-in bug report template?

Reply:
i has some questions based on built in bug template but has added questions based on what our internal needs are